nautobot/dcim/tests/test_filters.py

```diff
@@ -1422,60 +1418,74 @@ class ManufacturerTestCase(FilterTestCases.NameSlugFilterTestCase):
     def setUpTestData(cls):
         common_test_data(cls)
 
-        devices = Device.objects.all()
-
-        manufacturers = (
-            Manufacturer.objects.create(name="Manufacturer 4", slug="manufacturer-4", description="A"),
-            Manufacturer.objects.create(name="Manufacturer 5", slug="manufacturer-5", description="B"),
-            Manufacturer.objects.create(name="Manufacturer 6", slug="manufacturer-6", description="C"),
-        )
+        devices = list(Device.objects.all()[:3])
+        cls.manufacturers = list(Manufacturer.objects.exclude(description="")[:3])
 
-        InventoryItem.objects.create(device=devices[0], name="Inventory Item 1", manufacturer=manufacturers[0])
-        InventoryItem.objects.create(device=devices[1], name="Inventory Item 2", manufacturer=manufacturers[1])
-        InventoryItem.objects.create(device=devices[2], name="Inventory Item 3", manufacturer=manufacturers[2])
+        InventoryItem.objects.create(device=devices[0], name="Inventory Item 1", manufacturer=cls.manufacturers[0])
+        InventoryItem.objects.create(device=devices[1], name="Inventory Item 2", manufacturer=cls.manufacturers[1])
+        InventoryItem.objects.create(device=devices[2], name="Inventory Item 3", manufacturer=cls.manufacturers[2])
 
     def test_description(self):
-        params = {"description": ["A", "B"]}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        descriptions = [m.description for m in self.manufacturers[:2]]
+        params = {"description": descriptions}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), len(descriptions))
 
     def test_inventory_items(self):
         inventory_items = InventoryItem.objects.all()[:2]
         params = {"inventory_items": [inventory_items[0].pk, inventory_items[1].pk]}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), len(inventory_items))
 
     def test_has_inventory_items(self):
         with self.subTest():
             params = {"has_inventory_items": True}
-            self.assertEqual(self.filterset(params, self.queryset).qs.count(), 3)
+            self.assertEqual(
+                self.filterset(params, self.queryset).qs.count(),
+                self.queryset.exclude(inventory_items__isnull=True).count(),
+            )
```

Comment:
`assertQuerysetEqual` for these various filter tests, please.

Reply:
Oof. I was doing equivalence to the hardcoded count tests from before.